docs/enterprise/reference/manage.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Manage
+lang: en-US
+sidebarDepth: 2
+meta:
+    - name: keywords
+      content: configuration options settings Pomerium enterprise console
+---
+
+# Manage
+
+## Routes
+
+A Route provides access to a service through Pomerium.
+
+
+### General
+
+The **General** tab defines the route path, both from the internet and to the internal service, and the policies attached. Note that policies enforced on a Namespace the Route resides in will also be applied.
```

Comment:
It would be cool if we linked back to the concept page with an anchor anytime we mentioned any core concept like `Namespace` or `Route`. Basically any-proper nouning would be nice to have an internal link IMHO

Reply:
Agreed on the cross-linking, but I think it might make more sense to link to that concept's heading in the Reference section from the Reference section, and cross-link from the Concepts page to other headings in Concepts. Each term that has an anchor in the Concepts page could be linked once at the start of that concept's reference doc.

My thinking here is to keep the context the same for the user. That is to say, a user reading through the concepts page will likely be newer to Pomerium, and a user looking through Reference documentation is likely to already grasp the concepts, and be more interested in seeing how they interact with each other practically within the Console.

Reply:
Something like [this](https://github.com/pomerium/pomerium/commit/e9972cf4b9810367e2fd44baadf755ceb8ea7cd1)

Reply:
Agree.